packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -151,48 +165,45 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
 
   /**
    * Returns the issuers of attestations for a phoneNumber/account combo
-   * @param phoneNumber Phone Number
-   * @param account Account
+   * @param identifier Attestation identifier (e.g. phone hash)
+   * @param account Address of the account
    */
-  getAttestationIssuers = proxyCall(
-    this.contract.methods.getAttestationIssuers,
-    tupleParser(PhoneNumberUtils.getPhoneHash, (x: string) => x)
-  )
+  getAttestationIssuers = proxyCall(this.contract.methods.getAttestationIssuers)
 
   /**
    * Returns the attestation state of a phone number/account/issuer tuple
-   * @param phoneNumber Phone Number
-   * @param account Account
+   * @param identifier Attestation identifier (e.g. phone hash)
+   * @param account Address of the account
    */
   getAttestationState: (
-    phoneNumber: string,
+    identifier: string,
     account: Address,
     issuer: Address
   ) => Promise<AttestationStateForIssuer> = proxyCall(
     this.contract.methods.getAttestationState,
-    tupleParser(PhoneNumberUtils.getPhoneHash, stringIdentity, stringIdentity),
-    (state) => ({ attestationState: parseInt(state[0], 10) })
+    undefined,
```

Comment:
Why are we passing in `undefined` as input? I'm not sure I follow.

Reply:
because this param (the input pre-parser) is not needed but the next one (output pre-parser) is needed